third_party/2and3/click/types.pyi

```diff
@@ -79,47 +79,27 @@ class FloatRange(FloatParamType):
     ) -> None:
         ...
 
-
-class File(ParamType):
+class File(ParamType, Generic[AnyStr]):
     def __init__(
```

Comment:
This is a similar situation to the AtomicWriter above.

Reply:
I'd also just go with `IO[Any]` for now.